modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -27,10 +26,10 @@ spec:
     spec:
       template:
         spec:
-          storageClassName: <storage_class_name>
+          storageClassName: <appropriate_class_name>
 ...
 ----
-**  Set a default storage class by running the following commands:
+. Set the new default storage class by running the following commands:
 +
 [source,terminal]
 ----
```

Comment:
This is unrelated to your update, but these terminal commands are missing the "$" prompts, if you wouldn't adding them either now or in a future PR?

Reply:
Done.